docs/apache-airflow-providers-postgres/index.rst

```diff
@@ -65,7 +65,7 @@ Package apache-airflow-providers-postgres
 `PostgreSQL <https://www.postgresql.org/>`__
 
 
-Release: 5.2.0
+Release: 6.0.0
```

Comment:
I think this file is generated automatically so you shouldn't edit it

Reply:
@RosterIn is right. see note just abiive:


>   Please, only add notes to the Changelog just below the "Changelog" header when there are some breaking changes
>    and you want to add an explanation to the users on how they are supposed to deal with them.
>   The changelog is updated and maintained semi-automatically by release manager.


Also I think the change is not breaking. Dependency change is not a breaking change, bumping min version is not breaking either, and the behaviour is not going to change either

Reply:
Thanks @RosterIn , I've removed the changes to index.rst.

Reply:
@potiuk I considered it a breaking change because it will break for some people, because it introduces new operating system dependencies, that were not necessarily present before. For example on Debian-based distros, libpq-dev is necessary. This is the case for the slime docker images, for example airflow:slim-2.3.3-python3.10 .

Reply:
This is not what we consider as breaking. Dependency changes are not introducing. Our SemVer rules state (see readme)

Airflow: SemVer rules apply to core airflow only (excludes any changes to providers). Changing limits for versions of Airflow dependencies is not a breaking change on its own.

And while this is not changing the version but other 'variant' of the same dependency, this only affect how you install airflow. And it is not breaking the 'use' of Airflow which is the important factor for breaking/non breaking decision.

The main thing here that changes that are only affecting the way how you install Airfloe are not really 'breaking' any user code or ways how you use airflow. If for any reason you cannot install Airflow new version, you just cannot install it and you don't break anything. Once you installed it, it works without changes. So it does not break anything.

If you follow the logic of 'dependencies are breaking changed' then it basically would mean than any release where any dependency change is breaking - because it might cause troubles when installing Airflow or providers.